packages/machine/src/models/app-instance.ts

```diff
@@ -106,7 +106,7 @@ export class AppInstance {
     const latestState = json.latestState;
     for (const key in latestState) {
       // @ts-ignore
-      if (latestState[key]["_hex"]) {
+      if (latestState[key] && latestState[key]["_hex"]) {
```

Comment:
This was added because `playerName` was `undefined` and is also not part of the app state. It is UI state but for some reason it was added into the same object that gets passed to the install protocol's `initialState` parameter.